src/openrct2-cli/openrct2-cli.vcxproj

```diff
@@ -34,6 +34,18 @@
     <TargetName>openrct2-cli</TargetName>
     <LibraryPath>$(SolutionDir)bin;$(LibraryPath)</LibraryPath>
   </PropertyGroup>
+  <PropertyGroup Label="Vcpkg" Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
```

Comment:
Can't these conditions simply be removed? The value of `VcpkgEnabled` doesn't change.

Reply:
Visual Studio did that, I changed this configuration via the Properties.

Reply:
I tested it without conditions, seems to be fine.